AnkiDroid/src/main/java/com/ichi2/anki/Whiteboard.java

```diff
@@ -100,6 +103,22 @@ public Whiteboard(AbstractFlashcardViewer cardViewer, boolean inverted, boolean
         createBitmap();
         mPath = new Path();
         mBitmapPaint = new Paint(Paint.DITHER_FLAG);
+
+        // selecting pen color to draw
+        mColorPalette = (LinearLayout) cardViewer.findViewById(R.id.whiteboard_pen_color);
+
+        Button penColorWhite = (Button) cardViewer.findViewById(R.id.pen_color_white);
```

Comment:
Implementer's choice: Can be shortened to: 
```suggestion
        ((Button) cardViewer.findViewById(R.id.pen_color_white)).setOnClickListener(this::onClick);
```